src/main/java/me/rigamortis/seppuku/impl/module/render/HudModule.java

```diff
@@ -19,6 +20,7 @@
  */
 public final class HudModule extends Module {
 
+    public final Value<HudModule.Mode> mode = new Value<HudModule.Mode>("Sorting Mode", new String[]{"Sorting", "sort"}, "Changes arraylist sorting.", HudModule.Mode.LENGTH);
```

Comment:
This isnt the right place to put this value. There isn't a very good way to give hud components values right now and I'm not sure what the best way to do this would be. I would make a seperate module specifically to hold the value(s) for the ArrayList component.